### ISSUE TITLE
Midiori ver 1.1 not detected

### ISSUE THREAD

**Issue by suk2013** [tdaede/midiori#5]
Hi, 
I don't know where to post my questions below and I couldn't find the Discussions link so I opened an issue.
I built a Midiori ver 1.1 and I tested it on my x68K computer but it wasn't detected. Do I need to program anything to the components? I wonder if it's because some components not soldered properly or I used the incorrect components. I attached list of components that I used. The capacitor value that I used was 0.1uF (I assumed u1 is 0.1uF).  I saw other closed issue was suggested to use db.x program but I have no idea where to get this. Any help would be appreciated.
[Midiori Parts.pdf](https://github.com/user-attachments/files/16590473/Midiori.Parts.pdf)

**Comment by tdaede** [comment]
There is nothing to be programmed. It has been a while since I built a 1.1 but it should just work. Did you get your YM3802-X from a trustworthy source? Fakes are very common.

**Comment by suk2013** [comment]
Thank you for your reply. I got it from utsource. It seems a trustworthy source.

**Comment by suk2013** [comment]
Thomas, I was able to get my midiori card to work. I soldered manually all components on another pcb. Previosly I used solder paste and hot air gun. It was my first time using this method. I'm not sure if it's because of my soldering work error or there is an issue with my edge connector. It wasn't beveled so I used filing tool to bevel it. The bevel wasn't equal on both top and bottom. The new pcb I beveled both sides more equally. I'm happy that I can get it to work. Thank you so much to you and Mike.
I have some questions: 
1. Is it easy to modify the pcb design with Kicad from  midi din connectors to mini din? (I'm not an electronics engineer and I've never designed a pcb). 
2. I assume the TRS connector can only connect with TRS to midi adapter/cable, correct? I can't use it with TRS to RCA cable, right?
3. What's JP2 for? I assume Open is interrupt 2 and Short is interrupt 4. I set mine to Short.
4. How do I test the card if it's working correctly? So far I can only test with games directly with my x68k computer.
